nservicebus/outbox/index.md

```diff
@@ -78,9 +78,45 @@ In order to gradually convert an entire system from the DTC to the Outbox:
 
 WARNING: When verifying outbox functionality, it can sometimes be helpful to temporarily [stop the MSDTC Service](https://technet.microsoft.com/en-us/library/cc770732.aspx). This ensures that the Outbox is working as expected, and no other resources are enlisting in distributed transactions.
 
+## Concurrency
+
+Outbox does not prevent invoking handlers or sagas more then once, it only prevents that any changes configured in persistence is only committed once. If an endpoint allows concurrent message processing or is scaled out it can happen that the same message is processed more then once. If you are using non-transactional resources then make sure processing is idempotent. If this is unwanted then make sure you only run a single endpoint and have [configured the endpoint to process messages sequentially by setting the maximum concurrency to 1](/nservicebus/operations/tuning.md#tuning-concurrency).
```

Comment:
I'm not sure what the first two sentences really add to this section, is that related to concurrency?

> If an endpoint allows concurrent message processing or is scaled out it can happen that the same message is processed more then once.

this seems to mix up retries and concurrency a bit? the same message can be processed multiple times due to retries, but how can the same message be processed multiple times concurrently?

more **than** once